packages/core/src/scp-cf/destination-accessor.ts

```diff
@@ -207,25 +216,37 @@ export async function getDestinationFromDestinationService(
 }
 
 function tryDestinationForServiceBinding(name: string): Destination | undefined {
+  logger.info('Attempting to retrieve destination from service binding.');
   try {
-    return destinationForServiceBinding(name);
+    const destination = destinationForServiceBinding(name);
+    logger.info('Sucessfully retrieved destination from service binding.');
+    return destination;
   } catch (error) {
-    logger.warn(error.message);
-    logger.warn("If you're not using SAP Extension Factory, you can ignore this warning.");
+    logger.info(error.message);
+    logger.info('Could not retrieve destination from service binding.');
+    logger.info('If you are not using SAP Extension Factory, this information probably does not concern you.');
     return undefined;
   }
 }
 
 function tryDestinationFromEnv(name: string): Destination | undefined {
+  logger.info('Attempting to retrieve destination from environment variable.');
```

Comment:
I think this method should be aligned with the `tryDestinationForServiceBinding` in case something goes wrong it should be catched and go on. Assume somebody made a type in the `env` and it is not parsable JSON or the cast  in `getDestinationsFromEnv` fails for some other reason, as far as I can see the exception will fly up all the way and we do not get any information what was the content of the varialbe. So perhaps a pokemon catch and then log the value of the env variable for investigation what went wrong?

Reply:
I agree with @mr-flannery that the logging of the env can cause problems, but then I would perhaps add a log statement like `Error in parsing the env variable - log into cf and use the command XYZ to access the process.env.destination variable`.

Reply:
No. 1 good point, I will take care of that.
No. 2. I don't know what a good suggestions would be, `cf env` will print all env variables, but this is supposedly a lot to find the issue. Probably it would be best to go to the cockpit and check it there, but that's nothing I would like to describe in the logs.

Reply:
hm ok. I did not know the details. I thought there is a slim `CF CLI` command for getting the env varialbes you are interested in. If this is not the case just take care of the flying up.